project/MimaExcludes.scala

```diff
@@ -174,6 +174,35 @@ object MimaExcludes {
     ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.sql.types.ObjectType"),
     ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.sql.types.ObjectType$"),
 
+    // SPARK-44496: Move Interfaces needed by SCSC to sql/api.
+    ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.sql.Encoder"),
```

Comment:
@amaliujia Apologies for the late comment. Why were these `ProblemFilters` placed in `defaultExcludes` instead of `v35excludes`? Will they never be involved in the Mima check again?

Reply:
@LuciferYang 

The intention here is to exclude these from spark 3.4 Mima check and I expected once we release Spark 3.5, these will be deleted?

Reply:
If this is the case, they should be put into `v35excludes` for future unified cleanup, rather than being placed into `defaultExcludes`. The items placed into `defaultExcludes` generally represent the rules that will never participate in the mima check. I've submitted a PR to move the similar cases to v35excludes : https://github.com/apache/spark/pull/42912